README.md

```diff
@@ -1,8 +1,134 @@
-# duckbase
+duckbase
+=========================
 React + Redux Wrapper for Firebase
 
 ![Duck Taunt](ducktaunt.jpg)
 
-### License
+## Features
+- Automatic binding to the data your components need
+- Automatic unbinding for data no longer needed
```

Comment:
Not super important, but you have in the line above "the data your ..." so maybe this line should match? Or the one above it? idk